net-nds/openldap/openldap-2.6.6-r2.ebuild

```diff
@@ -373,6 +373,67 @@ src_prepare() {
 		-e '/MKDIR.*.(DESTDIR)\/run/d' \
 		servers/slapd/Makefile.in || die 'adjusting slapd Makefile.in failed'
 
+	if ! use debug; then
+		sed -r \
+			-e '/^    ldap_dump_(connection|requests_and_responses);$/d' \
+			-i libraries/libldap/ldap.map || die 'could not remove debug symbols'
+	fi
+
+	if ! use sasl; then
```

Comment:
Man, this is gnarly. Can't we just do `append-ldflags $(test-flags-CCLD -Wl,--undefined-version)`?

Reply:
FreeBSD said this can be fixed by add ld flags, but they choose the way in this patch.

Reply:
The issues were exposed by a change in lld's defaults, but it's not like it's UB or something. The lld maintainers changed it because they felt that it would expose QA issues in projects, but in reality, it ends up being a pain for projects with a lot of build options. IMO it's more maintainable to pass the flag to restore the old default than have a large sed (or several).